scripts/generate_pipeline_files.py

```diff
@@ -0,0 +1,169 @@
+import itertools
```

Comment:
Please add the license header at the top.

Reply:
added here: 
https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/412/commits/9080e14727d66ba4533b6d3dfb8bf577b2ce5939#diff-b760de9bd0fdeea47e69e074deb04cb2ef5604a0ae70ef0fa58b04121d206318R1-R11